executors/ovhapi/README.md

```diff
@@ -0,0 +1,63 @@
+# Venom - Executor OVHAPI
+
+Step to test ovh api
+
+Use case: you software need to make call to OVH api.<br>
+You will need ovh credentials to make api call. Please follow this tutorial to get all needed keys : <br>
+FR : https://www.ovh.com/fr/g934.premiers_pas_avec_lapi <br>
+EN : https://api.ovh.com/g934.first_step_with_api
+
+## Input
+In your yaml file, you can use:
+
+```
+  - method optional, default value : GET
+  - endpoint mandatory : eg "ovh-eu"
+  - applicationKey mandatory
+  - applicationSecret mandatory
+  - consumerKey: mandatory
+  - path mandatory, example "/me"
+  - needAuth optional : make authenfitication
+  - body optional
+  - bodyFile optional
+```
+
+```yaml
+
+name: Title of TestSuite
+testcases:
+
+- name: GET /me testcase
+  steps:
+  - type: ovhapi
+    endpoint: ovh-eu
```

Comment:
why there is no applicationKey, applicationSecret and consumerKey in this sample.
In fact it could be nice to be able to set an OVH Context with those values to avoid copy an paste them on all steps